dashboard/src/components/jobs/parameters/CompareJobParams.vue

```diff
@@ -43,7 +42,30 @@
           </label>
           <br />
         </div>
-        <div class="my-1">
+        <div v-if="containsPredicted">
+          <p>What is the time resolution of your data?</p>
+          <div class="ml-1 mt-1">
+            <input
+              type="radio"
+              id="hourly-resolution"
+              v-model="timeResolution"
+              value="leHourly"
+            />
+            <label for="hourly-resolution">My data is hourly or better.</label>
+            <br />
+            <input
+              disabled="true"
+              type="radio"
+              id="monthly-resolution"
+              v-model="timeResolution"
+              value="monthly"
+            />
+            <label class="greyed" for="monthly-resolution">
```

Comment:
whoops, I meant grey the hourly. I think monthly will be available sooner

Reply:
Gotcha. duh, that makes more sense.